Tensile/Source/client/main.cpp

```diff
@@ -640,16 +737,17 @@ int main(int argc, const char* argv[])
 
                                 for(int j = 0; j < enq; j++)
                                 {
+                                    int flush_index = (j + i*enq + 1) % flush_count;
```

Comment:
Is there any reason we use + 1 here?
If I understand the code correctly, I am feeling we do validation for [0].
But, if we start from 1, we might not store the result to [0] if the number of iteration is less than flush buffer num.
Don't we need to worry about that?

Reply:
validation is done for [0] but only duiring the warm up loop. We are starting hot loop from 1 to not reuse cache values from warm up.

Reply:
Can NumWarmUp be 0?

Reply:
I have not heard from you about this.
Does this work with NumWarmUp=0?